src/db_state.rs

```diff
@@ -225,6 +226,9 @@ pub(crate) struct CoreDbState {
     pub(crate) compacted: Vec<SortedRun>,
     pub(crate) next_wal_sst_id: u64,
     pub(crate) last_compacted_wal_sst_id: u64,
+    /// the `last_clock_tick` includes all data in L0 and below --
```

Comment:
Perhaps we could move this comment over to manifest.fbs? Would it be worth changing the name to something like `last_l0_clock_tick` or would that be overkill?

Reply:
+1 to `last_l0_clock_tick` both here and in manifest.